scenarios/reproducers/3-nodes.yml

```diff
@@ -0,0 +1,66 @@
+---
+cifmw_install_yamls_repo: /home/cjeanner/work/github/openstack-k8s-operators/install_yamls
+cifmw_basedir: "{{ ansible_user_dir }}/ci-framework-data"
+cifmw_path: "{{ ansible_user_dir }}/.crc/bin:{{ ansible_user_dir }}/.crc/bin/oc:{{ ansible_user_dir }}/bin:{{ ansible_env.PATH }}"
+cifmw_reproducer_repositories:
+  - src: "{{ playbook_dir }}"
+    dest: "src/github.com/openstack-k8s-operators/"
+  - src: "{{ cifmw_install_yamls_repo }}"
+    dest: "src/github.com/openstack-k8s-operators/"
+cifmw_libvirt_manager_configuration:
+  vms:
+    compute:
+      amount: 1
+      image_url: "{{ cifmw_discovered_image_url }}"
+      sha256_image_name: "{{ cifmw_discovered_hash }}"
+      image_local_dir: "{{ cifmw_basedir }}/images/"
+      disk_file_name: "centos-stream-9.qcow2"
+      disksize: 50
+      memory: 8
+      cpus: 4
+      nets:
+        - default
+        - crc
+    controller:
+      image_url: "{{ cifmw_discovered_image_url }}"
+      sha256_image_name: "{{ cifmw_discovered_hash }}"
+      image_local_dir: "{{ cifmw_basedir }}/images/"
+      disk_file_name: "centos-stream-9.qcow2"
+      disksize: 50
+      memory: 8
+      cpus: 4
+      nets:
+        - default
+        - crc
+    crc:
+      admin_user: core
+      image_local_dir: "{{ ansible_user_dir }}/.crc/machines/crc/"
+      disk_file_name: "crc.qcow2"
+      disksize: 150
+      memory: 32
+      cpus: 24
+      nets:
+        - default
+        - crc
+  networks:
+    default: |-
+      <network>
+        <name>default</name>
+        <forward mode='nat'/>
+        <bridge name='virbr0' stp='on' delay='0'/>
+        <mac address='52:54:00:6a:f2:dc'/>
+        <ip family='ipv4' address='192.168.100.1' prefix='24'>
+          <dhcp>
+            <range start='192.168.100.10' end='192.168.100.100'/>
+          </dhcp>
+        </ip>
+      </network>
+    crc: |-
+      <network>
+        <name>crc</name>
+        <forward mode='nat'/>
+        <bridge name='crc' stp='on' delay='0'/>
+        <mac address='52:54:00:fd:be:d0'/>
+        <ip family='ipv4' address='192.168.122.1' prefix='24'>
```

Comment:
This is the default ip range for the default libvirt network.
Should we use different range (192.168.130.1) for crc or use the default network ?

Reply:
so I've seen crc using .122 but also .130 range - it seems to depend on the version of crc. Thing is, in install_yamls, many assumptions are done about the used range, and many things are hardcoded to .122 range (routes, for instance).

Reply:
I create a crc image by just setup and stop.

It leads to I have :
cat /etc/hosts |grep crc
192.168.130.11 api.crc.testing canary-openshift-ingress-canary.apps-crc.testing console-openshift-console.apps-crc.testing default-route-openshift-image-registry.apps-crc.testing downloads-openshift-console.apps-crc.testing oauth-openshift.apps-crc.testing


Which step supposed to make either the existing record working,
or updating it to a new one ?

Reply:
the record on the hypervisor won't be used actually. So I don't think we should temper with it.